packages/mint-components/src/components/sqm-referral-table/sqm-referral-table.tsx

```diff
@@ -40,33 +40,46 @@ export class ReferralTable {
 
   /** @uiName Show Referred by user in table  */
   @Prop() showReferrer?: boolean = false;
+  
+  /** @uiName Hide Columns (Mobile View)  */
```

Comment:
We need to document how this works to hide multiple columns, also should probably start from 1 not 0 - people setting up programs are likely not as comfortable with indexing from 0